users/migrations/0001_initial.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.1.7 on 2021-03-22 11:39
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
```

Comment:
From what I understand, this dependency has something to do with the AbstractBaseUser class.
I think that there's a many to many connection somewhere there with django.auth,
so `migrate` has to run on django.auth first, and this is why there's the dependency.

"Since `User` inherits from `AbstractUser`, it will inherit `user_permissions` which is a many-to-many field with `auth.Permission`. (It will also inherit a many-to-many field with `auth.Group`)."
an answer from "tim-mccurrach" to a question posted on
https://stackoverflow.com/questions/65540414/django-claiming-inconsistent-migration-history-on-first-migration

Reply:
Another source I found:
https://stackoverflow.com/questions/51084875/why-does-initial-migration-have-dependencies

Reply:
I'll approve it if you tell me that if someone clones the repo and runs 'migrate' that it will work without an error

Reply:
@tamirho tried to clone the repo from my fork yesterday and ran `vagrant up` (which ran `migrate` beacuse of `bootstrap.sh`) and successfully created a superuser, logged into the admin page and ran tests.

Reply:
I can confirm this too, just tried it and it works without an error. (Added screenshots to the slack channel)